src/stores/sessions.js

```diff
@@ -36,6 +40,56 @@ export const useSessionsStore = defineStore('sessions', {
           }
         }
       }
+    },
+    async fetchToken () {
```

Comment:
can we call this: fetchSessionToken? just to distinguish it from any other API token.